### ISSUE TITLE
One of the ChangeWindows script keeps the light font of the dark mode

### ISSUE THREAD

**Issue by Radical4ever** [t1m0thyj/WDD-scripts#21]
The issue is pretty self explanatory, but apart from this they tray icon window is on dark mode and the classic context menu is on dark mode too, explorer window too, but if you close&open the window it changes to light mode. the only workaround its to go to task manager and reboot explorer.exe.
thank you
![imagen](https://user-images.githubusercontent.com/83252104/188272395-062df9f6-0f3f-405a-8353-8d1444cbb505.png)

**Comment by leostudiooo** [comment]
Yes this problem is really annoying. Another solution is to open settings-personlization-theme and change the color from light to dark and to light again. Can't figure why it would happen.

**Comment by Radical4ever** [comment]
I use the automatic accent color to work around with but it isnt a 100% solution

**Comment by bigplayer-ai** [comment]
same
